spring-cloud-dataflow-composed-task-runner/pom.xml

```diff
@@ -24,6 +24,7 @@
 	<packaging>jar</packaging>
 	<properties>
 		<plexus.utils.version>3.3.0</plexus.utils.version>
+		<spring.cloud.common.security>1.6.0-SNAPSHOT</spring.cloud.common.security>
```

Comment:
remove as this should already be in parent pom and only in parent pom.

Reply:
Removed